liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -974,22 +974,24 @@ public void installFeatures(File configFile, File serverDir) {
                         Set<String> existingFeaturesCopy = new HashSet<String>(existingFeatures);
                         existingFeaturesCopy.removeAll(featuresCopy);
                         if (!existingFeaturesCopy.isEmpty()) {
-                            log.info("Configuration features have been removed");
+                            log.info("Configuration features have been removed: " + existingFeaturesCopy);
                             existingFeatures.removeAll(existingFeaturesCopy);
                         }
                     }
 
                     // check if features have been added and install new features
                     if (!features.isEmpty()) {
-                        log.info("Configuration features have been added");
+                        log.info("Configuration features have been added: " + features);
+                        // pass all features to install feature
```

Comment:
what is the reason we are now passing all features to install feature?

Reply:
Sorry this comment was confusing, adjusted to clarify we are passing all _new_ features to install feature